48 combine built in parsers into one document (#152)
humio/docs.humio.com: content/tutorial/_index.md

### 4.1 Extra: Custom Parsers

> While Humio has built in support for the most popular logging formats (e.g. AccessLog, JSON), and can rip out almost anything with the [kv]({{< relref "../operations-guide/parsers/built-in-parsers/#kv" >}}) parser, you may have special needs for your custom application logs. Fortunately, Humio allows you to create your own [custom parsers]({{< relref "../operations-guide/parsers/creating-a-parser.md" >}}).

Reviewer: ```suggestion
While Humio has built in support for the most popular logging formats (e.g. AccessLog, JSON), and can rip out almost anything with the [kv]({{< ref "built-in-parsers.md#kv" >}}) parser, you may have special needs for your custom application logs. Fortunately, Humio allows you to create your own [custom parsers]({{< ref "creating-a-parser.md" >}}).
```

> While Humio has built in support for the most popular logging formats (e.g. AccessLog, JSON), and can rip out almost anything with the [`kv`]({{< relref "../operations-guide/parsers/built-in-parsers/_index.md/#kv" >}}) parser, you may have special needs for your custom application logs. Fortunately, Humio allows you to create your own [custom parsers]({{< relref "../operations-guide/parsers/creating-a-parser.md" >}}).

Reviewer: The ``[`kv`]({{< relref "../operations-guide/parsers/built-in-parsers/_index.md/#kv" >}})`` link is broken

## 5. Next Steps

- [Language Syntax]({{< relref "developer-resources/language-syntax/_index.md" >}})
- [Query Function Reference]({{< relref "developer-resources/query-functions/_index.md" >}})

> - [Creating Custom Parsers]({{< relref "operations-guide/parsers/creating-a-parser.md" >}})

Reviewer: ```suggestion
- [Creating Custom Parsers]({{< ref "creating-a-parser.md" >}})
```

Author: @mwl Yeah, those changes were related to another issue 105 I was working on. If I had made them in the other branch, I would have had conflicts when I went to merge, as the changes were to a parser file that was deleted in this branch. So I went ahead and made them here in the combined document.

> - [Creating Custom Parsers]({{< relref "../operations-guide/parsers/creating-a-parser.md" >}})

Reviewer: ```suggestion
- [Creating Custom Parsers]({{< ref "creating-a-parser.md" >}})
```
